src/ripple/overlay/impl/Cluster.cpp

```diff
@@ -113,7 +113,8 @@ Cluster::load(Section const& nodes)
             return false;
         }
 
-        auto const id = parseBase58<PublicKey>(TokenType::NodePublic, match[1]);
+        auto const id = parseBase58<PublicKey>(
+            TokenType::NodePublic, std::string_view{match[1].str()});
```

Comment:
Are you sure this cast is necessary? Doesn't `str()` return a `std::string` which is implicitly convertible to `std::string_view`?

Reply:
thanks, fixed in https://github.com/XRPLF/rippled/commit/85e001dcb21516ad0d711e60af70fc9141f29162